frontend/src/components/spatialChat/chatInputBox/chatInputBox.test.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
For this test, the biggest issue was with ts-ignore. There are some test entities which have to be ts-ignored. Followed the same pattern as used in the `town-settings.test.tsx`